frontend/src/pages/RunDetailsRouter.tsx

```diff
@@ -58,35 +36,11 @@ export default function RunDetailsRouter(props: RunDetailsProps) {
     {},
   );
 
-  // Get Recurring run ID (if it is existed) for retrieve recurring run detail (PipelineSpec IR).
```

Comment:
Will this break `V1 run created by recurring run` functionality?

Reply:
Simply answer is no.

Actually, **v1 never use the recurring run id here**. Previously, existing run can be either v1 or v2 run. We use `getRecurringRunId` helper to **get v2 recurring run id** and use it to retrieve the pipeline spec in recurring run body. However, when I work on the run integration, v2 run is no longer has resource reference. The helper should be removed at that moment rather than mis-embedding with v1 run. Therefore, it is just a misconception that the change might affect v1 run.